README.md

```diff
@@ -46,13 +46,25 @@ The page will reload if you make edits.
 
 ### App configuration
 
-Currently, app can be configured by setting the following environment variables:
+Currently, app can be configured by setting the following environment variables in .env file in home directory of the project.
 
-- `REACT_APP_BACKEND_URL` - URL of the API server (default: `http://localhost:5000`)
+#### Setting communication with backend
+
+ `REACT_APP_BACKEND_URL` - URL of the API server. e.g.
```

Comment:
UI can be deployed on different machine (with different IP) than a backend. During build phase the UI can be configured to talk to given backend instance via `REACT_APP_BACKEND_URL`